src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs

```diff
@@ -33,6 +33,16 @@ public class GenevaExporterOptions
     /// </summary>
     public IEnumerable<string>? CustomFields { get; set; }
 
+    /// <summary>
+    /// Gets or sets WithResourceAttributes.
+    ///
+    /// WithResourceAttributes specifies which resource attribute fields should be sent to Geneva.
+    ///
+    /// Any resource attributes not in WithResourceAttributes are ignored.
+    /// If WithResourceAttributes is not provided, no resource attributes will be sent to Geneva.
+    /// </summary>
+    public IEnumerable<string>? WithResourceAttributes { get; set; }
```

Comment:
do clarify that this is replacement for prepopulatedfields, and prepopulatedfields should be removed before enabling this to avoid sending duplicate fields.

Reply:
Now I recall there is no concept of prepopulated in Rust

Reply:
prepopulated fields is not ignored if resourcefieldnames is specified